sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQuerySuite.scala

```diff
@@ -451,6 +451,17 @@ class StreamingQuerySuite extends StreamTest with BeforeAndAfter with Logging wi
     assert(progress.sources(0).numInputRows === 10)
   }
 
+  test("[SPARK-19690] stream join with aggregate batch query succeeds") {
```

Comment:
Can you move this to the StreamingAggregationSuite? Because that suite is closely related to this aggregation bug. And I would rename it to "SPARK-19690: do not convert batch aggregation in streaming query to streaming aggregation"